packages/MarkdownEditor-Core.package/MarkdownCodeBlock.class/instance/asText.st

```diff
@@ -0,0 +1,9 @@
+converting
+asText
+
+	| code result |
+	code := self smalltalkStyler styledTextFor: self codeContent asText.
```

Comment:
After reading the `STTextstyler` class comment, `styledTextFor:` is wrong here. It also does formatting (which could be dangerous if it actually modifies the text) whereas `style:` has a slight different interface but should be correct.
I tried to make a quick test but it got late and I think this issue is complex enough for a pair.
Or, if you don't care about the test, we can just replace this line ^^

Rough STR:

1. Open _Preference Browser_
2. Enable `syntaxHighlightingAsYouTypeLeftArrowAssignment`
3. Open _Markdown Editor_ and type a code block, including `:=`
  _Your image should be frozen_

Reply:
We should definitely fix this! Regression test would be nice (as discussed face to face).

Reply:
The solution involved the use of `privateStyle:` as `style:` styles a _copy_ of the string and then informing its view. As we have no adequate view (and we found it strange to make a code block a view), we just call `privateStyle:`...